streamer/pulp/streamer/server.py

```diff
@@ -129,7 +159,7 @@ def _handle_get(self, request):
                 for entry in q_set.all():
                     logger.info('Trying URL: {url}'.format(url=entry.url))
                     try:
-                        last_report = self._download(entry, responder)
+                        last_report = self._download(request, entry, responder)
```

Comment:
Ok, thanks for chasing that down! This is a good time to make sure we fully understand how the streamer work, so I feel much better with this mystery solved.